app/components/datafruits-visuals.js

```diff
@@ -98,10 +127,11 @@ export default Component.extend({
             //mp4 exists, play it
             this.streamIsActive(name, "mp4");
           } else {
-            console.log("No stream found"); // eslint-disable-line no-console
-            later(()=> {
-              this.fetchStream();
-            }, 15000);
+            console.log("No stream found");
+            //later(()=> {
+            yield timeout(15000);
```

Comment:
Running into issues with this, you can't call `yield` inside an arrow function.
`/c/Users/mcfir/src/datafruits/datafruits13/components/datafruits-visuals.js: Unexpected reserved word 'yield' (132:12)
`